lms/templates/commerce/receipt.underscore

```diff
@@ -1,7 +1,10 @@
 <div class="wrapper-content-main payment-confirmation-step">
   <article class="content-main">
     <h3 class="title">
-      <%= interpolate(gettext( "Thank you! We have received your payment for %s"), ["<span class='course_name_placeholder'></span>"]) %>
+      <%= interpolate(
+            gettext( "Thank you! We have received your payment for %(course_name)s."),
```

Comment:
@clintonb gettext doesn't work in the same line as interpolate in underscore templates. Please be sure that you're properly testing your features for i18n.